source/core/core.vcxproj

```diff
@@ -190,6 +190,7 @@
     <ClInclude Include="slang-memory-arena.h" />
     <ClInclude Include="slang-object-scope-manager.h" />
     <ClInclude Include="slang-random-generator.h" />
+    <ClInclude Include="slang-render-api-util.h" />
```

Comment:
I'm not entirely happy with this code moving into `core`, but if it solves a near-term problem I guess we can live with it.

Eventually we should figure out how to keep all the 3D-API-related code out of `core`.